src/visualization.jl

```diff
@@ -161,8 +161,12 @@ function D3Trees.D3Tree(nodes::Vector{Dict{String, Any}}; title="Julia D3Tree",
             rel_q = (n["q"]-min_q)/(max_q-min_q)
             color = weighted_color_mean(rel_q, colorant"green", colorant"red")
             style[i] = "stroke:#$(hex(color))"
-            w = 20.0*sqrt(n["n"]/n["parent_n"])
-            link_style[i] = "stroke-width:$(w)px"
+            try # Some nodes may not have "parent_n"
+                w = 20.0*sqrt(n["n"]/n["parent_n"])
+                link_style[i] = "stroke-width:$(w)px"
+            catch
+                nothing
```

Comment:
In general, `try` blocks with a "do-nothing" catch should be avoided since they can hide errors. Instead, this should be handled with a more specific `if` branch or something similar like `get(n, "parent_n", n["n"])`.

Reply:
For sure, I used this when trying to figure out what was happening (I didn't notice I had `enable_tree_vis=false` and I was getting exception from `n["parent_n"]` because there was no such key) and forgot to remove this from the commit. I removed the try block.